### ISSUE TITLE
Does IUC needs a website?

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/tools-iuc#3987]
We now have the new hub and a new fancy static-side builder. We could use this infrastructure (or a similar one) to generate a IUC website. 
If generic enough, we could integrate it into the CI system and it could be used in galaxy-p and other repos as well.

I think we have enough metadata to spin up a useful website for any of our tools. A few ideas are included in the mockup.

![IUC website](https://user-images.githubusercontent.com/469983/134767387-84ec531e-e7ec-4e3c-8f65-7fd324be8080.png)

No clue if we can easily integrate the rendered tool form, but getting the tool runs (from GRT?) is easy and could be done monthly on CI.

This could be the first step to finally write a paper about IUC.

Any comments or ideas?

**Comment by Slugger70** [comment]
I think it's an awesome idea. Especially if you can search tools etc.

**Comment by mvdbeek** [comment]
I think it's an interesting idea, but if we want to write a paper we can do this without a site like this, that should really not stop us given it isn't central to what the IUC does.

On the practical side I think it would be very challenging to show a "live" tool form since so much of the logic is evaluated on the backend, just name, description, long description seems like a good start.

**Comment by bgruening** [comment]
> I think it's an interesting idea, but if we want to write a paper we can do this without a site like this, that should really not stop us given it isn't central to what the IUC does.

Agreed, but we did not manage until now, so maybe that is a motivation :)

**Comment by bernt-matthias** [comment]
We just discussed this in the tools SIG meeting (starting with the observation that IWC has a shiny website).

But we agreed that the toolshed should be the desired website.

**Comment by bernt-matthias** [comment]
Ping @SaimMomin12
